sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -430,13 +424,13 @@ def clear(self):
 
   def _commit(self):
     if self._cleared:
-      self._state_handler.blocking_clear(self._state_key)
+      self._state_handler.clear(self._state_key, is_cached=True).get()
```

Comment:
Is this the call where we don't need to wait for a response?

Reply:
Yes, and no. We need to wait for the last response here. For example, if there is no following append, we'll have to wait on the clear. So best to safe the last returned future in a variable and call `.get()`on it here.